documentation/staging/content/faq/external-clients.md

```diff
@@ -197,7 +197,7 @@ Here are the steps:
 
 - In WebLogic, configure a custom channel for the T3 protocol that enables HTTP tunneling, and specifies an external address and port that correspond to the address and port that remote applications will use to access the load balancer. See [Adding a WebLogic custom channel](#adding-a-weblogic-custom-channel) for samples and details.
 
-- Set up a load balancer that redirects HTTP traffic to the custom channel. For more information on load balancers, see [Ingress]({{<relref "/userguide/managing-domains/ingress/_index.md">}}). If you're using OKE/OCI to host your Kubernetes cluster, also see [Using an OCI Load Balancer]({{<relref "/faq/oci-lb">}}).
```

Comment:
Should OKE be Oracle Container Engine for Kubernetes? https://www.oracle.com/cloud-native/container-engine-kubernetes/. Of course, the text behind that link uses "OCI" :)

Reply:
Hmmm, interesting. That doc uses Oracle Container Engine for Kubernetes (OKE) and Oracle Cloud Infrastructure (OCI) (OCI being a no-no). I can change OKE to Oracle Container Engine for Kubernetes. Should I do it globally?

Reply:
Is it really the rule, "thou shalt not use OCI"? It seems like other documents use the full name, introduce the acronym, and then use that.

Reply:
Here's the guidance/advice/**rule** I got from Antoinette O'SULLIVAN: Oracle Linux Cloud Native Environment (renamed Oracle Cloud Native Environment some time back) or OLCNE (we don't abbreviate this product name) or OCI (we don't abbreviate this product name, since we have docs integrated in the core cloud docs (and core cloud docs don't use this abbreviation). I'm asking her now about OKE. Documentation guidelines do dictate following the legal/proper way of presenting product names.